store/storetest/job_store.go

```diff
@@ -44,13 +44,10 @@ func testJobSaveGet(t *testing.T, ss store.Store) {
 
 	defer ss.Job().Delete(job.Id)
 
-	if received, err := ss.Job().Get(job.Id); err != nil {
-		t.Fatal(err)
-	} else if received.Id != job.Id {
-		t.Fatal("received incorrect job after save")
-	} else if received.Data["Total"] != "12345" {
-		t.Fatal("data field was not retrieved successfully:", received.Data)
-	}
+	received, err := ss.Job().Get(job.Id)
```

Comment:
this should be `=`  not `:=`

Reply:
`received` is not declared, so `=` would cause an error